arm64v8/crate/docker-entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+set -ae
+
+# GC logging set to default value of path.logs
+CRATE_GC_LOG_DIR="/data/log"
+CRATE_HEAP_DUMP_PATH="/data/data"
+# Make sure directories exist as they are not automatically created
+# This needs to happen at runtime, as the directory could be mounted.
+mkdir -pv $CRATE_GC_LOG_DIR $CRATE_HEAP_DUMP_PATH
+
+# Special VM options for Java in Docker
+CRATE_JAVA_OPTS="-XX:+UnlockExperimentalVMOptions -Des.cgroups.hierarchy.override=/ $CRATE_JAVA_OPTS"
```

Comment:
Why do we need the `XX:+UnlockExperimentalVMOptions` ?

Reply:
not sure, I just copied it from the amd64 script. I will drop it.